lldb/include/lldb/Target/Target.h

```diff
@@ -258,6 +258,8 @@ class TargetProperties : public Properties {
 
   bool GetDebugUtilityExpression() const;
 
+  Args GetDebugInfoDURLs() const;
```

Comment:
General naming nit:

`debuginfod` is all lower-case, and capitalized as if it were a single word: `Debuginfod`.

Reply:
I *really* hate that, but that's clearly consistent with the rest of the code. I need to pronounce it as a single word, I think. "Dehbooginfawd"

Reply:
I tend to think of it as agglutanitive, like "blackbird."